requirements-docs-lock.txt

```diff
@@ -275,7 +275,7 @@ urllib3==1.26.19 ; python_version < "3.10" \
     # via
     #   -r requirements-docs.txt
     #   requests
-zipp==3.19.2 \
-    --hash=sha256:bf1dcf6450f873a13e952a29504887c89e6de7506209e5b1bcc3460135d4de19 \
-    --hash=sha256:f091755f667055f2d02b32c53771a7a6c8b47e1fdbc4b72a8b9072b3eef8015c
+zipp==3.19.1 \
```

Comment:
@jonathan343 This is something we may need to think about with the addition of the docs lock file. We're now out of sync between the dev-lock and the docs-lock. I spoke with Steve briefly about this for the CLI but we should figure out of we need to establish some for of sim-ship process when doing these updates.

Reply:
Also I'm realizing we don't have a good way to establish confidence in these changes from dependabot since we're not using the docs lock file in the GHA tests.